pennylane/optimize/adam.py

```diff
@@ -27,19 +27,19 @@ class AdamOptimizer(GradientDescentOptimizer):
     the momentum and velocity of a particle:
 
     .. math::
-        x^{(t+1)} = x^{(t)} - \eta^{(t+1)} \frac{a^{(t+1)}}{\sqrt{b^{(t+1)}} + \epsilon },
+        x^{(t+1)} = x^{(t)} - \eta^{(t)} \frac{a^{(t+1)}}{\sqrt{b^{(t+1)}} + \epsilon },
 
-    where the update rules for the three values are given by
+    where the update rules for the two moments are given by
 
     .. math::
-        a^{(t+1)} &= \frac{\beta_1 a^{(t)} + (1-\beta_1)\nabla f(x^{(t)})}{(1- \beta_1)},\\
-        b^{(t+1)} &= \frac{\beta_2 b^{(t)} + (1-\beta_2) ( \nabla f(x^{(t)}))^{\odot 2} }{(1- \beta_2)},\\
+        a^{(t+1)} &= \beta_1 a^{(t)} + (1-\beta_1) \nabla f(x^{(t)}),\\
+        b^{(t+1)} &= \beta_2 b^{(t)} + (1-\beta_2) ( \nabla f(x^{(t)}))^{\odot 2}.
         \eta^{(t+1)} &= \eta^{(t)} \frac{\sqrt{(1-\beta_2)}}{(1-\beta_1)}.
```

Comment:
I don't think we should define `eta` recursively.  It's instead a factor times the initial, unmodified stepsize.

Reply:
Good point, this is fixed now. The original docstring was defining it recursively which was confusing.